### ISSUE TITLE
一对实体存在多种关系（EPO）模型如何全部识别出来呢

### ISSUE THREAD

**Issue by lqq688** [Ricardokevins/Bert-In-Relation-Extraction#58]
一对实体存在多种关系（EPO）模型如何全部识别出来呢？{"rel": "歌手", "ent1": "张宇", "ent2": "离开", "text": "《离开》是由张宇谱曲，演唱"}，{"rel": "作曲", "ent1": "张宇", "ent2": "离开", "text": "《离开》是由张宇谱曲，演唱"}，对于这两个测试用例模型能分别正确预测出这两种关系吗

**Comment by Ricardokevins** [comment]
> 系（EPO）模型如何全部识别出来呢？{"rel": "歌手", "ent1": "张宇", "ent2": "离开", "text": "《离开》是由张宇谱曲，演唱"}，{"rel": "作曲", "ent1": "张宇", "ent2": "离开", "text"

你可以试一试

**Comment by lqq688** [comment]
模型测试是ent1+ent2+text输入到模型，最后的预测只取最大值 _, predicted = torch.max(logits.data, 1)，就是模型只能预测一种关系，比如：我测试时输入（ 张宇+离开+《离开》是由张宇谱曲，演唱。）模型只能预测出最大值为：演唱。但是我的测试案例是{"rel": "作曲", "ent1": "张宇", "ent2": "离开", "text": "《离开》是由张宇谱曲，演唱"}，说明就是预测错了。

**Comment by lqq688** [comment]
Source Text:  《离开》是由张宇谱曲，演唱
Entity1:  张宇  Entity2:  离开  Predict Relation:  歌手  True Relation:  作曲
Source Text:  《离开》是由张宇谱曲，演唱
Entity1:  张宇  Entity2:  离开  Predict Relation:  歌手  True Relation:  歌手
1   2   0.5

**Comment by lqq688** [comment]
如果训练集和测试集全部都是一对实体存在多种关系的情况，模型表现会非常差，因为他没有解决关系重叠的问题

**Comment by lqq688** [comment]
请问如何解决关系重叠的问题

**Comment by Ricardokevins** [comment]
你可以阅读最新的论文或者其他的最新解决方案
